plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryTableHandle.java

```diff
@@ -52,12 +50,6 @@ public BigQueryTableHandle(
         this.projectedColumns = requireNonNull(projectedColumns, "projectedColumns is null");
     }
 
-    @Deprecated
```

Comment:
Removing this constructor seems unrelated to BigLake support. Please separate a commit.

Reply:
Moved to a separate commit.